fase_di_sviluppo/src/modules/calendar.js

```diff
@@ -1,26 +1,86 @@
-/**
- * Implements the Calendar class.
- * 
- * This file implements the class Calendar, which
- * represent a list of homework.
- * 
- * @file This file defines Calendar class.
- * @author Andrea Lanzani
- * Everyone who modifies this file MUST add his/her name
- * @version 1.0alpha
- */
-
-/**
- * @todo Implement this class.
- * @todo Write the documentation.
- * 
- * Class representing a calendar.
- */
-export class Calendar {
-  /**
-   * Create a calendar.
-   */
-  constructor() {
-
-  }
+/**
+ * Implements the Calendar class.
+ * 
+ * This file implements the class Calendar, which
+ * represent a list of homework.
+ * 
+ * @file This file defines Calendar class.
+ * @author Andrea Lanzani
+ * @author Francesco Colombo
+ * Everyone who modifies this file MUST add his/her name
+ * @version 1.0alpha
+ */
+
+/**
+ * @todo Implement this class.
+ * @todo Write the documentation.
+ * 
+ * Class representing a calendar.
+ */
+
+export class Calendar {
+
+  constructor() {
+        this._homework = null;
+  }
+
+  addHomework(homework){
+      
+    if(!homework.duration.isBusy()){
+        this.homework.push(homework);
+    }
+    else{
+      window.alert("Could not enter assignment");
+    }
+        
+  }
+
+  findHomework(title, subject){
+
+    let element;
+    if(title!=null)
+        element = this.find(title,"title");
+    else if(subject!= null)
+        element = this.find
+        (subject,"subject");
+
+    //element will appear somehow maybe by document.getElemetById().innerHtml or .value
+  }
+
+   find(type,propertis){
+
+    let isFind;
+    let i = 0;
+    isFind = false;

```

Comment:
Consiglio di rinominare la variabile in 'found'. Inoltre è del tutto inutile separare la dichiarazione e l'assegnamento
```suggestion
    let found = false;
```